reference/map-first-preprocessing.dita

```diff
@@ -12,6 +12,20 @@
       experimental feature, has been improved and is ready for use in many production scenarios. Map-first-preprocessing
       provides the same functionality as the default <codeph>preprocess</codeph>, but takes a different
     approach.</ph></shortdesc>
+  <prolog>
+    <metadata>
+      <keywords>
+        <indexterm>filter</indexterm>
+        <indexterm>preprocessing<indexterm>map first</indexterm></indexterm>
+        <indexterm>DITA 1.0</indexterm>
+        <indexterm>DITA 1.2</indexterm>
+        <indexterm>DITA 1.3</indexterm>
```

Comment:
While each of these spec versions are _mentioned_ in the topic, it isn't really about those spec versions, so these seem to create noise that distracts from topics that are actually about the specs.

Reply:
The entries do provide information about how maps are processed relative to those spec versions. The map first processing only really makes a lot of sense because of the additional features added in DITA 1.2 and 1.3. I think there's value to someone who is using an older version of DITA-OT and DITA and might learn about how newer DITA-OT and DITA versions could affect their workflows.